card/src/main/java/com/adyen/checkout/card/CardView.kt

```diff
@@ -195,13 +203,16 @@ class CardView @JvmOverloads constructor(context: Context, attrs: AttributeSet?
                 }
                 binding.textInputLayoutCardHolder.error = mLocalizedContext.getString(holderNameValidation.reason)
             }
-            val postalCodeValidation = it.postalCodeState.validation
+            val postalCodeValidation = it.addressState.postalCode.validation
             if (binding.textInputLayoutPostalCode.isVisible && postalCodeValidation is Validation.Invalid) {
                 if (!isErrorFocused) {
                     binding.textInputLayoutPostalCode.requestFocus()
                 }
                 binding.textInputLayoutPostalCode.error = mLocalizedContext.getString(postalCodeValidation.reason)
             }
+            if (binding.addressFormInput.isVisible && !it.addressState.isValid()) {
+                binding.addressFormInput.highlightValidationErrors(it.addressState)
```

Comment:
We should pass the `isErrorFocused` field to `addressFormInput.highlightValidationErrors` and use it to check the focus there, right?